frontendservice/src/views/specificShop.vue

```diff
@@ -155,8 +155,8 @@ export default {
         Cookie.set("tokenid", JSON.stringify(self.allCookieList), {
           expires: 1
         });
-        self.statusFlag = false;
-        self.flag = true;
+        self.isTokenActive = true;
```

Comment:
Didn't see the entire code, is this boolean flip intensional?

Reply:
yes priyanka.